doc/dev/MAINTAINERS_AND_CONTRIBUTORS.md

```diff
@@ -0,0 +1,25 @@
+# Maintainers and Contributors
+
+This document lists the Maintainers and Returning Contributors of the Project.
+
+## Maintainers
+
+Maintainers may add themselves once approved by the existing Maintainers as described in the [Governance document](./GOVERNANCE.md). By adding your name to this list you are agreeing to abide by and uphold the Project governance documents and to abide by all of the Organization's policies, including the [code of conduct](../../CODE_OF_CONDUCT.md).
+
+| **NAME**         | **Handle** |
+| ---------------- | ---------- |
+| Mathieu Westphal | mwestphal  |
+| Michael Migliore | Meakk      |
+
+## Returning Contributors
+
+Returning Contributors may add themselves once requirements have been checked be existing Maintainers.
+Maintainers may decide to list a returning contributor as inactive after a year without contributions.
+Contributors with a single contribution are not listed here but can be found in the git history.
+
+| **NAME** | **Handle** | **Active** |
```

Comment:
Maybe we should specify it's ok to stay anonymous and list the handle only?

Reply:
Good point